cli/ops/worker_host.rs

```diff
@@ -189,10 +127,11 @@ fn run_worker_thread(
     // TODO(bartlomieju): this thread should return result of event loop
     // that means that we should store JoinHandle to thread to ensure
     // that it actually terminates.
-    run_worker_loop(&mut rt, &mut worker).expect("Panic in event loop");
+    rt.block_on(worker).expect("Panic in event loop");
```

Comment:
use tokio_util::run_basic(worker)

Reply:
Doesn't make sense here, need runtime to load modules beforehand